### ISSUE TITLE
pa11y-ci: investigate ignoring new version banner

### ISSUE THREAD

**Issue by ashharrison90** [grafana/grafana#42233]
see https://raintank-corp.slack.com/archives/C036J5B39/p1637679302051500 for context
